decorate.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'nameable'
```

Comment:
- [ ] Kindly remove the import above, the decorator class should be imported in your person.rb so that when we run the code below, we can see the output in the terminal 👍🏿 
``` 
person = Person.new(22, 'maximilianus')
 p person.correct_name
 capitalizedPerson = CapitalizeDecorator.new(person)
p capitalizedPerson.correct_name
 capitalizedTrimmedPerson = TrimmerDecorator.new(capitalizedPerson)
 p capitalizedTrimmedPerson.correct_name
```